libevm/VM.h

```diff
@@ -69,14 +69,14 @@ class VM: public VMFace
 
 private:
 
-	u256* m_io_gas = 0;
+	u256* io_gas = 0;
+	uint64_t m_io_gas = 0;
```

Comment:
What does 'UB' mean?  But I thought with two's complement everything but division was the same operation.  Anyway, no need to revisit uint64 versus int64 for this PR.